.github/dependabot.yml

```diff
@@ -3,11 +3,19 @@
 
 version: 2
 updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
+      day: sunday
+    open-pull-requests-limit: 20
+    assignees:
+      - johnnymetz
```

Comment:
I'm happy to take responsibility for this, unless there's someone else who is better suited to handle this